libs/plugin/src/generators/convert-di-to-inject/generator.ts

```diff
@@ -221,10 +221,14 @@ export async function convertDiToInjectGenerator(
 							}
 						});
 					} else {
+						const propertyName = options.useESPrivateFieldNotation
+							? `#${name}`
+							: name;
+
 						targetClass.insertProperty(index, {
-							name,
+							name: propertyName,
 							initializer,
-							scope,
+							scope: options.useESPrivateFieldNotation ? null : scope,
```

Comment:
If the field is protected I don't think we should convert it to private. So I'd only convert private properties to use the # and keep the others as is.